.github/workflows/ci-crossdock.yml

```diff
@@ -43,9 +43,13 @@ jobs:
     - name: Install tools
       run: make install-ci
 
-    - uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+    # Conditionally setup QEMU only for arm64 Linux architecture
+    - name: Setup QEMU for arm64 Linux
+      if: runner.os == 'Linux' && runner.arch == 'arm64'
```

Comment:
This condition is always true, and the build is still going to go through all arch

Reply:
Noted.